fs/sync/sync.go

```diff
@@ -92,6 +93,7 @@ type syncCopyMove struct {
 	setDirModTimeMu        sync.Mutex             // protect setDirModTimeMu
 	setDirModTimes         []setDirModTime        // directories that need their modtime set
 	setDirModTimesMaxLevel int                    // max level of the directories to set
+	modifiedDirs           map[string]struct{}    // dirs with changed contents (if s.setDirModTimeAfter)
```

Comment:
~~This needs a mutex to protect it `modifiedDirsMu` - it will be accessed concurrently~~

Edit: I see you are using the `setDirModTimeMu` which is fine - either annotate the help with that info, or rename the variable `setDirModTimeDirs` or something like that to group it with `setDirModTimeMu`

Reply:
Made this change.